components/Buttons/src/MDCButton.m

```diff
@@ -207,18 +213,20 @@ - (void)commonMDCButtonInit {
   _shouldRaiseOnTouch = YES;
   _uppercaseTitle = YES;
   _userElevations = [NSMutableDictionary dictionary];
-  _backgroundColors = [NSMutableDictionary dictionary];
   _accessibilityLabelForState = [NSMutableDictionary dictionary];
 
-  _backgroundColors[@(UIControlStateNormal)] = MDCColorFromRGB(MDCButtonDefaultBackgroundColor);
+  if (!_backgroundColors) {
+    // _backgroundColors may have already been initialized by setting the backgroundColor setter.
+    _backgroundColors = [NSMutableDictionary dictionary];
+    _backgroundColors[@(UIControlStateNormal)] = MDCColorFromRGB(MDCButtonDefaultBackgroundColor);
```

Comment:
Should this be:
```
if (!_backgroundColor) {
  _backgroundColors = [NSMutableDictionary dictionary];
}
_backgroundColors[@(UIConrolStateNormal)] = MDCColorFromRGB(MDCColor);
```

Do we not need to [self updateBackgroundColor] as we are below?